src/assignments/scrabble-scorer.rst

```diff
@@ -327,12 +373,14 @@ Bonus Missions
 #. Score words spelled with blank tiles by adding ``' '`` to the
    ``newPointStructure`` object. The point value for a blank tile is ``0``.
 
-Submitting Your Work
----------------------
+.. TODO: what does submission look like now?
+
+.. Submitting Your Work
+.. --------------------
 
-#. From the address bar at the top of the browser window, copy the URL of the
-   repl.it that contains your solution.
-#. Go to the Graded Assignment #2 page in Canvas and click *Submit Assignment*.
-#. Paste the URL into the Website URL input.
-#. Click *Submit Assignment* again.
-#. Notify your TA that your assignment is ready to be graded.
+.. #. From the address bar at the top of the browser window, copy the URL of the
+..    repl.it that contains your solution.
+.. #. Go to the Graded Assignment #2 page in Canvas and click *Submit Assignment*.
+.. #. Paste the URL into the Website URL input.
+.. #. Click *Submit Assignment* again.
+.. #. Notify your TA that your assignment is ready to be graded.
```

Comment:
"If in the Instructor-Led Track, notify your TA that your assignment is ready to be graded" or something to that extent...